### PR TITLE
fix startup script

### PR DESCRIPTION
**Pull request by kcpevey**
Resolves bug with using startup scripts.

### CODE REVIEW

File: examples/example_update_repo.py [hunk 1]
```diff
@@ -3,14 +3,15 @@
 from ib_manifest_util import TEST_DATA_DIR
 from ib_manifest_util.update_repository import update_repo
 
-repo_dir = TEST_DATA_DIR.joinpath("sample_repo")
+repo_dir = Path("/Users/kcp/projects/metrostar/demo/singleuser")
```
Comment (iameskild) [inline examples/example_update_repo.py]: I think this needs to be reverted or updated.

File: examples/example_update_repo.py [hunk 2]
```diff
@@ -3,14 +3,15 @@
...
     dockerfile_version="9999",
-    local_env_path=repo_dir.joinpath("scripts", "local_channel_env.yaml"),
-    output_hardening_path=output_dir.joinpath("output_hardening_manifest.yaml"),
-    output_dockerfile_path=output_dir.joinpath("output_dockerfile"),
-    dockerfile_template_path=None,
+    # local_env_path=repo_dir.joinpath("scripts", "local_channel_env.yaml"),
+    startup_scripts_path="None",
+    # output_hardening_path=output_dir.joinpath("output_hardening_manifest.yaml"),
+    # output_dockerfile_path=output_dir.joinpath("output_dockerfile"),
+    # dockerfile_template_path=None,
```
Comment (iameskild) [inline examples/example_update_repo.py]: Are all of these meant to be commented out?

**Review by iameskild — APPROVED** [review]
Nice addition :)

I had two minor comments but otherwise, this is should be ready to merge.

File: examples/example_update_repo.py (hunk 1 above)
Comment (kcpevey) [reply examples/example_update_repo.py]: yep, that was accident

File: examples/example_update_repo.py (hunk 2 above)
Comment (kcpevey) [reply examples/example_update_repo.py]: accident, thanks!

**Review by iameskild — APPROVED** [review]

**Pull request merged**